SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -50,6 +56,73 @@ void GetSubobjects(UObject* Object, TArray<UObject*>& InSubobjects)
 	});
 }
 
+Worker_EntityId USpatialPackageMapClient::AllocateEntityIdForActor(AActor* Actor)
+{
+	check(Actor);
+
+	Worker_EntityId EntityId = NetDriver->EntityPool->GetNextEntityId();
```

Comment:
If we ever attempt this on a client we'll get a hard crash, worth guarding against?

Reply:
added check for NetDriver->IsServer before attempting to get an entity ID from the pool